tck/src/main/java/org/eclipse/microprofile/lra/tck/participant/api/LraController.java

```diff
@@ -159,22 +157,22 @@ public Response completeWork(@HeaderParam(LRA_HTTP_CONTEXT_HEADER) String lraId,
         }
 
         activity.setStatus(ParticipantStatus.Completed);
-        activity.setStatusUrl(String.format("%s/%s/activity/completed", context.getBaseUri(), lraId));
+        activity.setStatusUrl(String.format("%s/%s/activity/completed", context.getBaseUri(), lraId.toASCIIString()));
 
-        LOGGER.info(String.format("LRA id '%s' was completed", lraId));
+        LOGGER.info(String.format("LRA id '%s' was completed", lraId.toASCIIString()));
         return Response.ok(activity.getStatusUrl()).build();
     }
 
     @PUT
     @Path("/compensate")
     @Produces(MediaType.APPLICATION_JSON)
-    @Compensate(timeLimit = 0, timeUnit = ChronoUnit.SECONDS)
-    public Response compensateWork(@HeaderParam(LRA_HTTP_CONTEXT_HEADER) String lraId, String userData)
+    @Compensate(timeLimit = 0, timeUnit = ChronoUnit.SECONDS)  // O and ChronoUnit.SECONDS are the default, just to be explicit here.
```

Comment:
wouldn't it be better to remove timelimit here? If we change defaults in the future is very likely we can forget to update the comment. Also there are some timelimit tests for Compensate coming soon I think.